odl/diagnostics/examples.py

```diff
@@ -104,20 +102,20 @@ def vector_examples(space):
         stds = np.apply_along_axis(np.std, axis=0, arr=points)
 
         def element(fun):
-            return space.element(uspace.element(fun))
+            return space.element(fun)
 
         # Indicator function in first dimension
-        def _step_fun(*args):
-            z = np.zeros(_arg_shape(*args))
+        def _step_fun(x):
```

Comment:
Do all of these work now? Tried running the [example](https://github.com/odlgroup/odl/blob/issue-57__operator/examples/operator_test.py)?